packages/perseus/src/widgets/interactive-graphs/types.ts

```diff
@@ -12,3 +16,21 @@ export type MafsGraphProps<T extends InteractiveGraphState> = {
     graphState: T;
     dispatch: (action: InteractiveGraphAction) => unknown;
 };
+
+export type InteractiveGraphState = SegmentGraphState | LinearGraphState;
+
+export interface InteractiveGraphStateCommon {
+    hasBeenInteractedWith: boolean;
+    // range = [[xMin, xMax], [yMin, yMax]] in Cartesian units
+    range: [Interval, Interval];
```

Comment:
Do you think Typescript's ability to label tuple members would help here a bit? I know you can't use these labels to reference the values, but I do find it helps a bit as it removes some of the need for a comment like we have above. 

```suggestion
    range: [x: Interval, y: Interval];
```

Reply:
TIL!